tests/test_mirror_port_erspan.py

```diff
@@ -0,0 +1,951 @@
+            elif fv[0] == "SAI_MIRROR_SESSION_ATTR_DST_MAC_ADDRESS":
+                assert fv[1] == "88:88:88:88:88:88"
+
+        session_status= "1:"+format(mirror_entries[0])
+        # Verify port has mirror session enabled with session oid.
+        status = self.getPortAttr(dvs, src_oid1, 'SAI_PORT_ATTR_INGRESS_MIRROR_SESSION')
+        assert status == session_status
+        # Verify port has egress mirror session enabled with session oid.
+        session_status = self.getPortAttr(dvs, src_oid1, 'SAI_PORT_ATTR_EGRESS_MIRROR_SESSION')
+        assert status == session_status
+
+        # Verify port has mirror session enabled with session oid.
+        status = self.getPortAttr(dvs, src_oid2, 'SAI_PORT_ATTR_INGRESS_MIRROR_SESSION')
+        assert status == session_status
+        # Verify port has egress mirror session enabled with session oid.
+        session_status = self.getPortAttr(dvs, src_oid2, 'SAI_PORT_ATTR_EGRESS_MIRROR_SESSION')
+        assert status == session_status
+        # remove neighbor
+        self.remove_neighbor("PortChannel008", "11.11.11.11")
+        assert self.get_mirror_session_state(session)["status"] == "inactive"
+
+        # remove ip address
+        self.remove_ip_address("PortChannel008", "11.11.11.0/24")
+        assert self.get_mirror_session_state(session)["status"] == "inactive"
+
+        #verify mirror config is removed from LAG source ports.
+        session_status= "1:"+format(mirror_entries[0])
+        # Verify port has mirror session enabled with session oid.
+        status = self.getPortAttr(dvs, src_oid1, 'SAI_PORT_ATTR_INGRESS_MIRROR_SESSION')
+        assert status == None or status == "0:null"
+        # Verify port has egress mirror session enabled with session oid.
+        status = self.getPortAttr(dvs, src_oid1, 'SAI_PORT_ATTR_EGRESS_MIRROR_SESSION')
+        assert status == None or status == "0:null"
+
+        # Verify port has mirror session enabled with session oid.
+        status = self.getPortAttr(dvs, src_oid2, 'SAI_PORT_ATTR_INGRESS_MIRROR_SESSION')
+        assert status == None or status == "0:null"
+        # Verify port has egress mirror session enabled with session oid.
+        session_status = self.getPortAttr(dvs, src_oid2, 'SAI_PORT_ATTR_EGRESS_MIRROR_SESSION')
+        assert status == None or status == "0:null"
+
+        # bring down port channel and port channel member
+        self.set_interface_status(dvs, "PortChannel008", "down")
+        self.set_interface_status(dvs, "Ethernet88", "down")
+
+        # remove port channel member; remove port channel
+        self.remove_port_channel_member("008", "Ethernet88")
+        self.remove_port_channel(dvs, "008")
+        self.remove_port_channel_member("001", src_port1)
+        self.remove_port_channel_member("001", src_port2)
+        self.remove_port_channel(dvs, "001")
+
+        # remove mirror session
+        self.remove_mirror_session(session)
```

Comment:
Missing newline at the end of this file.

Reply:
Done.